canonical-websites/www.ubuntu.com
# Flatten terms section

## Done

- Flattened legal terms-and-policies section
- Updated navigation
- Updated redirects
- Added some special redirects requested by legal team

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- look at the files in the /legal directory and compare to this [spreadsheet](https://docs.google.com/spreadsheets/d/12kuuGgwRqN3k7wlRc4QKIuZifjvxbynizO_LEjKjBq4/edit#gid=0)
* test the link [/legal/terms](http://0.0.0.0:8001/legal/terms) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/terms) redirects also test that [/terms](http://0.0.0.0:8001/terms) redirects as well
* test the link [/legal/intellectual-propery-policy](http://0.0.0.0:8001/legal/intellectual-propery-policy) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/intellectual-property-policy) redirects also test that [/ip-policy](http://0.0.0.0:8001/ip-policy) redirects as well
* test the link [/legal/intellectual-property-policy/2013-05-14](http://0.0.0.0:8001/legal/intellectual-property-policy/2013-05-14) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/intellectual-property-policy/2013-05-14) redirects
* test the link [/legal/online-account-terms](http://0.0.0.0:8001/legal/online-account-terms) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/online-account-terms) redirects
* test the link [/legal/terms-of-service](http://0.0.0.0:8001/legal/terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/terms-of-service) redirects
* test the link [/legal/confidentiality-agreement](http://0.0.0.0:8001/legal/confidentiality-agreement) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/confidentiality-agreement) redirects
* test the link [/legal/developer-terms-and-conditions](http://0.0.0.0:8001/legal/developer-terms-and-conditions) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/developer-terms-and-conditions) redirects
* test the link [/legal/launchpad-terms-of-service](http://0.0.0.0:8001/legal/launchpad-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/launchpad-terms-of-service) redirects
* test the link [/legal/livepatch-terms-of-service](http://0.0.0.0:8001/legal/livepatch-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/livepatch-terms-of-service) redirects
* test the link [/legal/jaas-beta-terms-of-service](http://0.0.0.0:8001/legal/jaas-beta-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/jaas-beta-terms-of-service) redirects
* test the link [/legal/systems-information-notice](http://0.0.0.0:8001/legal/systems-information-notice) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/systems-information-notice) redirects
* test the link [/legal/font-licence](http://0.0.0.0:8001/legal/font-licence) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/font-licence) redirects
* test the link [/legal/privacy-policy](http://0.0.0.0:8001/legal/privacy-policy) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/privacy-policy) redirects also test that [/privacy-policy](http://0.0.0.0:8001/privacy-policy) redirects as well

* test the link [/legal/terms](http://0.0.0.0:8001/legal/terms) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/terms) redirects also test that [/terms](http://0.0.0.0:8001/terms) redirects as well
--
* test the link [/legal/intellectual-propery-policy](http://0.0.0.0:8001/legal/intellectual-propery-policy) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/intellectual-property-policy) redirects also test that [/ip-policy](http://0.0.0.0:8001/ip-policy) redirects as well
* test the link [/legal/intellectual-property-policy/2013-05-14](http://0.0.0.0:8001/legal/intellectual-property-policy/2013-05-14) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/intellectual-property-policy/2013-05-14) redirects
* test the link [/legal/online-account-terms](http://0.0.0.0:8001/legal/online-account-terms) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/online-account-terms) redirects
* test the link [/legal/terms-of-service](http://0.0.0.0:8001/legal/terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/terms-of-service) redirects
* test the link [/legal/confidentiality-agreement](http://0.0.0.0:8001/legal/confidentiality-agreement) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/confidentiality-agreement) redirects
* test the link [/legal/developer-terms-and-conditions](http://0.0.0.0:8001/legal/developer-terms-and-conditions) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/developer-terms-and-conditions) redirects
* test the link [/legal/launchpad-terms-of-service](http://0.0.0.0:8001/legal/launchpad-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/launchpad-terms-of-service) redirects
* test the link [/legal/livepatch-terms-of-service](http://0.0.0.0:8001/legal/livepatch-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/livepatch-terms-of-service) redirects
* test the link [/legal/jaas-beta-terms-of-service](http://0.0.0.0:8001/legal/jaas-beta-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/jaas-beta-terms-of-service) redirects
* test the link [/legal/systems-information-notice](http://0.0.0.0:8001/legal/systems-information-notice) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/systems-information-notice) redirects
* test the link [/legal/font-licence](http://0.0.0.0:8001/legal/font-licence) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/font-licence) redirects
* test the link [/legal/font-licence/differences](http://0.0.0.0:8001/legal/font-licence/differences) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/font-licence/ofl-1.1-ufl-1.0.diff) redirects
* test the link [/legal/font-licence/faq](http://0.0.0.0:8001/legal/font-licence/faq) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/font-licence/faq) redirects
* test the link [/legal/privacy-policy](http://0.0.0.0:8001/legal/privacy-policy) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/privacy-policy) redirects also test that [/privacy-policy](http://0.0.0.0:8001/privacy-policy) redirects as well



## Issue / Card

Fixes #4197
